### PR TITLE
Added a setup.py script to test and run PEP8 checker against code

### LINKED ISSUE #36: Add a setup.py file for running tests and beautifying/linting
We need a setup.py script for automating some of our testing and formatting.

### PR DESCRIPTION
**Pull request by ericjperry**
* Added a setup.py script with tasks for testing all code and running a PEP8 style checker against the code.
* Removed the OBE test.sh script.
* Added a preliminary README with instructions for testing and style checking (we need to beef up the README big time with docs).
* Reformatted all the code to get rid of PEP8 warnings (I'm ignoring the multiple empty lines rule between classes/methods because it's annoying, for now).

I pushed all the reformatting stuff in a different commit, so if we want to back that out it should be easy. I can also change the rules and stuff if we want to do that (the most annoying one that I left was line length, which produced some kind of gross looking stuff, but I'll let you guys be the judges).

@KrisSiegel @sapanshah 

Resolves #36.

### CODE REVIEW

**Comment by sapanshah** [comment]
:+1:

File: test.sh [hunk 1]
```diff
@@ -1,3 +0,0 @@
-#!/bin/bash
```
Comment (KrisSiegel) [inline test.sh]: RIP :sob:

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+
+from setuptools import setup
+
+setup(
+    name='Bakula',
+    version='0.0.1',
```
Comment (KrisSiegel) [inline setup.py]: <insert long reason why this should be 1.0.0>

:trollface:

**Comment by KrisSiegel** [comment]
:+1:

**Pull request merged**